### PR TITLE
Development

### LINKED ISSUE #22: Reorganize routes
AFAIK now route to the debug panel is `/debug/viewer` and `/debug` just works as api.
I suggest to restructure these routes:
* `/debug` - panel
* `/debug/api` - api. It can be conflict with panel routes when we do routes to selected panel for example as `/debug/logger`. Then we cannot add panel with name `api`. But we can make a little workaround `/debug/panel/logger`.

### PR DESCRIPTION
**Pull request by rustamwin**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️/❌
| New feature?  | ✔️/❌
| Breaks BC?    | ✔️/❌
| Fixed issues  | Fix #22, #18, #17, #20, #19, #14

### CODE REVIEW

**Comment by samdark** [comment]
Looks 👍 so far.

File: resources/views/index.html [hunk 1]
```diff
@@ -5,22 +5,115 @@
...
+                            fill-rule: evenodd;
+                            clip-rule: evenodd;
+                            fill: #F18A2A;
+                        }
+
+                        .st3 {
+                            fill: #7FB93C;
+                        }
+                    </style>
+                    <g id="Слой-1" serif:id="–°–ї–Њ–є 1">
```
Comment (samdark) [inline resources/views/index.html:39]: I think that should be cleaned up.

File: resources/views/index.html [hunk 2]
```diff
@@ -5,22 +5,115 @@
...
+        <div class="yii-debug-toolbar__block">
+            <a href="#" title="">PHP
+                <span class="yii-debug-toolbar__label" data-key="phpVersion"></span>
+            </a>
+        </div>
+        <div class="yii-debug-toolbar__block">
+            <a href="#" title="Response status: 200">Status
+                <span class="yii-debug-toolbar__label yii-debug-toolbar__label_success" data-key="responseStatusCode">200</span>
+            </a>
+            <a href="#" title="Total request processing time was 15.17 ms">Time
```
Comment (samdark) [inline resources/views/index.html:67]: 15.17 ms? Is there a moment when this is visible before data is loaded? If so, it is better to use something more meaningless like "Loading" or just zero.

File: resources/views/index.html [hunk 3]
```diff
@@ -56,20 +149,61 @@
...
+            for (let i = 0; i < panelLinks.length; ++i) {
+                if (panelLinks[i].href.includes(ev.state.panel)) {
+                    panelLinks[i].click();
+                    break;
+                }
+            }
+        })
         document.addEventListener('click', event => {
             if (event.target.matches('.panel-link')) {
+                console.log(event);
```
Comment (samdark) [inline resources/views/index.html:199]: Debug info should be removed.

File: resources/views/index.html [hunk 4]
```diff
@@ -126,77 +223,160 @@
...
+                    if (div.innerHTML.includes('<table')) {
+                        div.classList.add('table-responsive')
+                    }
+                    document.querySelector(".router-view").appendChild(div);
+                    event.target.disabled = false;
+                },
+                error: (xhr) => {
+                    event.target.disabled = false;
+                    if (xhr.status === 404) {
+                        document.querySelector(".router-view").innerHTML = xhr.responseText;
```
Comment (samdark) [inline resources/views/index.html:289]: `document.querySelector(".router-view")` could be moved out of `if`.

File: resources/views/index.html (hunk 2 above)
Comment (rustamwin) [reply resources/views/index.html]: Leftover :)

File: resources/views/index.html (hunk 1 above)
Comment (rustamwin) [reply resources/views/index.html]: Done.

**Pull request merged**